.circleci/config.yml

```diff
@@ -24,8 +24,13 @@ jobs:
     - checkout
     # Workaround so that /go/bin is writable by the circleci user
     - run: sudo mkdir -p /go/bin && sudo chown -R circleci /go/bin
-    # Run unit tests, TODO
-    #- run: go test ./...
+    # Run unit tests.
+    - run:
+        name: unit tests
+        command: make unit-test-in-docker
+        # command: $GO test -mod=vendor $($GO list ./... | grep -v /e2e | grep -v /pkg/dm | grep -v pkg/runtime/containerd)
+        # environment:
+        #   - GO: "sudo /usr/local/go/bin/go"
```

Comment:
Tried running the unit tests on host but `go list` didn't seem to work with the go on the host. https://circleci.com/gh/weaveworks/ignite/1342?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

If running go test on host directly with sudo works, containerd can be installed to run the containerd client tests. This is excluded when running in container.
We can also try running containerd in a test container and run the test against it.

Reply:
Moved the tests to TravisCI that has containerd installed for e2e tests.